src/main/scala/com/datastax/gatling/plugin/request/CqlRequestAction.scala

```diff
@@ -69,43 +69,54 @@ class CqlRequestAction(val name: String,
       ThroughputVerifier.checkForGatlingOverloading(session, gatlingTimingSource)
       GatlingResponseTime.startedByGatling(session, gatlingTimingSource)
     }
-    val stmt = dseAttributes.statement.buildFromSession(session)
+    try {
+      val stmt = dseAttributes.statement.buildFromSession(session)
 
-    stmt.onFailure(err => {
-      val responseTime: ResponseTime = responseTimeBuilder.build()
-      val logUuid = UUID.randomUUID.toString
-      val tagString = if (session.groupHierarchy.nonEmpty) session.groupHierarchy.mkString("/") + "/" + dseAttributes.tag else dseAttributes.tag
+      stmt.onFailure(err => {
+        handleFailure(session, responseTimeBuilder, err)
+      })
 
-      statsEngine.logResponse(session, name, responseTime.toGatlingResponseTimings, KO, None,
-        Some(s"$tagString - Preparing: ${err.take(50)}"), List(responseTime.latencyIn(MICROSECONDS), "PRE", logUuid))
+      stmt.onSuccess({ stmt =>
+        // global options
+        dseAttributes.cl.map(stmt.setConsistencyLevel)
+        dseAttributes.dynamicCl.map(f => stmt.setConsistencyLevel(f()))
```

Comment:
What's this?

Reply:
that's me not merging my branches properly...  It's from the other PR